packages/assistify-help-request/server/hooks/searchHooks.js

```diff
@@ -0,0 +1,21 @@
+/* global SystemLogger */
+function onSpotlighStartHandler(item) {
+	SystemLogger.debug('onSpotlightStart', item.text);
+}
+
+function onSpotlighEndHandler(item) {
+	SystemLogger.debug('onSpotlightEnd', item.text);
+	if (item.result && item.result.rooms) {
+		const subscripedChannels = RocketChat.models.Subscriptions.findByUserId(Meteor.userId()).fetch();
```

Comment:
@lobo87 this will return subscriptions, not rooms

Reply:
this is indeed correct, i completely forgot to commit the last changes which has a mapping function to _room in there.